include/notcurses/notcurses.h

```diff
@@ -1641,6 +1642,7 @@ typedef struct nccapabilities {
   bool halfblocks;// we assume halfblocks, but some are known to lack them
   bool quadrants; // do we have (good, vetted) Unicode 1 quadrant support?
   bool sextants;  // do we have (good, vetted) Unicode 13 sextant support?
+  bool octants;   // do we have (good, vetted) Unicode 16 octant support?
```

Comment:
i don't see where we're ever setting this capability

Reply:
Yes, we'll have to examine terminal emulators to see whether/when they support this.

Reply:
.nodnod, so right now it's never going to be used. that's fine for this PR.